mr_adc_compute.py

```diff
@@ -223,7 +226,9 @@ def compute_trans_properties(mr_adc, E, U):
     U = np.array(U)
     T = np.dot(T, U.T)
 
-    spec_intensity = np.sum(T**2, axis=0)
+    #TODO: Discuss about it
+    # spec_intensity = np.sum(T**2, axis=0)
+    spec_intensity = 2.0 * np.sum(T**2, axis=0)
```

Comment:
What should we discuss here? Do spec factors match spin–orbital code? In the restricted SR-ADC implementation, we have to renormalize eigenvectors for spin–adapted spec factors to match the spin–orbital ones. I would expect the same should be done here

Reply:
I see. My approach solves the spec factors but not the eigenvectors. I will work on the implementation as done in SR-ADC.